src/coreclr/debug/createdump/crashinfo.cpp

```diff
@@ -182,7 +182,7 @@ CrashInfo::GatherCrashInfo(DumpType* dumpType)
     }
 #endif
     // Load and initialize DAC interfaces
-    if (!InitializeDAC())
+    if (!InitializeDAC(dumpType))
```

Comment:
I'd still say if we grab a non-full dump and then we fail to load the dac we should either notify through a warn, or we should fail.